### ISSUE TITLE
Insider reflow validation: Focus indicator is not visible on the left nav links during keyboard navigation with Screen readers

### ISSUE THREAD

**Issue by iamrafan** [microsoft/accessibility-insights-web#3095]
## Describe the bug

When navigating through the left nav in assessment with screen readers on, the focus indicator on the links is not visible after activating any of the links with keyboard. Once the focus indicator disappears it is not visible even when navigating through the list of test/requirements using an arrow key. This issue exists in the following browser/AT combinations

Browser | AT
-- | --
Chrome | NVDA
Edge | Narrator
Edge | JAWS

**To Reproduce**

1. Run assessment on 'https://www.washington.edu/accesscomputing/AU/before.html'
2. Use one of the above browser/AT combination to navigate to the left nav
3. Activate one of the links
4. Observe that the black focus ring disappears and is not visible hereon even when using arrow keys to navigate to other links though the screen readers keep announcing the link where the focus moves to.

## Expected behavior

After activation of a link, the focus indicator should be visible on the current link as well as the link navigated to, using the arrow key

### Keyboard navigation without AT
![Issue-1 1](https://user-images.githubusercontent.com/22068434/87700412-2080ad80-c74b-11ea-8865-b8b438559e15.gif)

### Keyboard navigation with Chrome/NVDA
![Issue-1 2](https://user-images.githubusercontent.com/22068434/87700421-24143480-c74b-11ea-9cc0-afd5f688776c.gif)

## Context
- OS Name: Windows 10, Version: 1903, Build 18362.476
- AI-Web Version & Environment: Insider 2.19.0 
- Browser Version: Google Chrome 84.0.4147.89, Microsoft Edge 83.0.478.64
- Target Page: https://www.washington.edu/accesscomputing/AU/before.html

**Comment by pownkel** [comment]
This seems to be an issue with the office fabric nav component. I've filed[ this issue ](https://github.com/microsoft/fluentui/issues/14089)with their team.

**Comment by ferBonnin** [comment]
@iamrafan OF team mentions this is a screen reader issue. Could we investigate if this has already being filed with the ATs and if there is something we could do to show a focus indicator ourselves

**Comment by iamrafan** [comment]
@ferBonnin This issue is related to https://github.com/microsoft/fluentui/issues/11793 filed by us

**Comment by karanbirsingh** [comment]
@ferBonnin looks like OF team says their fix is released https://github.com/microsoft/fluentui/issues/11793#issuecomment-820980950

**Comment by ferBonnin** [comment]
thanks @karanbirsingh. @iamrafan can you please check the fix? 
will remove the blocked tag after investigation

**Comment by iamrafan** [comment]
@ferBonnin The fix did not address the scenario for which we filed the bug, I am going to ask them to reopen the issue

**Comment by iamrafan** [comment]
@ferBonnin The external bug https://github.com/microsoft/fluentui/issues/11793 that we filed has been fixed. Here are further details about the validation: 

```NVDA Version 2021.1```
```JAWS Professional Edition Version 2020.2006.12 ```
```Google Chrome Version 92.0.4515.131```
```Fluent UI React 8.28.0```

## Screenshots

### Keyboard navigation with Chrome/NVDA
![NVDA](https://user-images.githubusercontent.com/22068434/129300579-c9f3e6fd-8c2f-4154-a6f4-95ba2cc8b228.gif)



### Keyboard navigation with Chrome/JAWS
![JAWS](https://user-images.githubusercontent.com/22068434/129300620-f9fac0dc-6b90-467e-9736-38692b98bf37.gif)

**Comment by iamrafan** [comment]
@ferBonnin The external bug fix has not resolved this issue, in fact I see new issues when navigation through the links. I shall start an email thread to discuss them further.
